js/services/api/pot-lucks.js

```diff
@@ -1,15 +1,64 @@
 import api from './base';
 
-const getPotLucks = () => {
-  return api.get('/potLucks');
+const getUserPotLucks = (userId) => {
+  return api.get(`potLuckDetails/${userId}`)
+    .then((result) => {
+      return result;
+    })
+    .catch((error) => {
+      console.log(error);
+    });
+};
+
+const getPotLuck = (potLuckId) => {
+  return api.get(`/potLucks/${potLuckId}`)
+    .then((result) => {
+      return result;
+    })
+    .catch((error) => {
+      console.log(error);
+    });
 };
 
 const create = (data) => {
-  return api.change('/potLucks', data);
-  // TODO add other inserts for child objects
+  // create potLuck and potLuckIndex objects
+  // TODO other potLuck components
+  const potLuck = {};
+  potLuck[data.userId] = true;
+  potLuck.title = data.title;
+  potLuck.theme = data.theme;
+  potLuck.eventDate = data.eventDate;
+  potLuck.arriveTime = data.arriveTime;
+  potLuck.serveTime = data.serveTime;
+  potLuck.location = data.location;
+  potLuck.description = data.description;
+
+  const potLuckIndex = {};
+  potLuckIndex.isHost = true;
+  potLuckIndex.eventDate = data.eventDate;
+  potLuckIndex.title = data.title;
+  potLuckIndex.description = data.description;
+  potLuckIndex.isNew = true;
```

Comment:
It's generally a bit cleaner to initialize the object w/ data rather than mutating an empty object:

```js
const potLuckIndex = {
  isHost: true,
  ...etc
}
```